ui-search/src/main/java/app/tivi/home/search/Search.kt

```diff
@@ -0,0 +1,200 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.tivi.home.search
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.Icon
+import androidx.compose.foundation.Text
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.preferredHeight
+import androidx.compose.foundation.layout.preferredWidth
+import androidx.compose.foundation.lazy.LazyColumnFor
+import androidx.compose.material.AmbientEmphasisLevels
+import androidx.compose.material.IconButton
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.ProvideEmphasis
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Clear
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.savedinstancestate.savedInstanceState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.onSizeChanged
+import androidx.compose.ui.platform.DensityAmbient
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import app.tivi.common.compose.PosterCard
+import app.tivi.common.compose.statusBarsPadding
+import app.tivi.data.entities.ShowTmdbImage
+import app.tivi.data.entities.TiviShow
+import app.tivi.data.resultentities.ShowDetailed
+
+@OptIn(ExperimentalFoundationApi::class, ExperimentalAnimationApi::class)
+@Composable
+fun Search(
+    state: SearchViewState,
+    actioner: (SearchAction) -> Unit
+) {
+    Box(Modifier.fillMaxSize()) {
+        val searchBarHeight = remember { mutableStateOf(0) }
+
+        SearchList(
+            results = state.searchResults,
+            contentPadding = PaddingValues(
+                top = with(DensityAmbient.current) { searchBarHeight.value.toDp() }
```

Comment:
Couldn't this use the `Insets.toPaddingValues` fun?

Reply:
Nah, it needs to be the status bar height + search bar.

Reply:
Ahhh yeah.